New package: AstroProp v0.3.0

JuliaRegistrator opened the pull request:
- Registering package: AstroProp
- Repository: https://github.com/GenAstro/Epicycle
- Created by: @stevenhughes73
- Version: v0.3.0
- Commit: f2260a25c418b9797f7c9db36883ba1cd5856fb6
- Reviewed by: @stevenhughes73
- Reference: https://github.com/GenAstro/Epicycle/commit/f2260a25c418b9797f7c9db36883ba1cd5856fb6#commitcomment-171467386
- Description: An application for space systems with a nod to the giants who came before us and to how far we’ve come. 
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
AstroProp is part of the Epicycle space mission design and navigation system.
Initial release - orbit propagation algorithms and integrators.

README - https://github.com/GenAstro/Epicycle/blob/main/AstroProp/README.md
docs - https://genastro.github.io/Epicycle/AstroProp/dev/

This package is part of a larger monorepo. The original PR removed the deps to avoid sequential registration and three day timers. This registration contains all deps, which only depends on EpicycleBase which is already merged (see here https://github.com/JuliaRegistries/General/pull/142542#event-21019565512)
`````
<!-- END RELEASE NOTES -->

<!-- 80cc8f6b60f354a8c40306cc5f8995f38a93dc181bbd7c02f357b084bc064489dcf37a94c705f01e834931e0ac74d0b2a65e35755cfa1e2ca1b5323d7687b0eecfde0a776d7d1055cac216e406399c293d346f804bfedd021f6a13a955cfd054f943a0ff3aed9f467e74023e71bcaa0972dd137c55fa8e281b2b6f07da53a1bb8c1f0c6638c36e640c1149b7a2577d8d7b027d0cab466b35a3f0a1cfc5d807ae33e291206ba5d1b9944bd569a880b6c7115f129051b52bb7fa8da8e4da7ff3a2b1120ef85a9620c52fea35fab57862466b5f417669d6137047cb61a067d91d4b -->

goerz commented:
Please comment the link to the new PR after it's been submitted

stevenhughes73 commented:
@goerz  I created the new branch and registered from there... here is the branch

https://github.com/GenAstro/Epicycle/commit/49b64883ffb8bf4be214ea94512aba08c5e83425

But oddly, it did not create a new PR, it updated this one.

goerz commented:
> But oddly, it did not create a new PR, it updated this one.

If you're not changing the package name or version, that's exactly what should happen.

That's also how this will ultimately resolve when you retrigger after all dependencies have been registered

JuliaRegistrator committed:
New package: AstroProp v0.3.0

UUID: 0e2365da-becf-4b4f-af1d-8bfff9db196c
Repo: https://github.com/GenAstro/Epicycle.git
Tree: 0023972a6c57bce31895b1a0d6d2b855934c685e

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
